lib/assertions/is-array-buffer.test.js

```diff
@@ -71,3 +71,42 @@ describe("assert.isArrayBuffer", function() {
         });
     });
 });
+
+describe("refute.isArrayBuffer", function() {
+    context("when called with an ArrayBuffer instance", function() {
+        it("should fail", function() {
+            var actual;
+            try {
+                referee.refute.isArrayBuffer(new global.ArrayBuffer(8));
+            } catch (error) {
```

Comment:
You could use [assert.throws](https://nodejs.org/api/assert.html#assert_assert_throws_fn_error_message) here. If you want to keep the individual asserts for the different aspects of the error, you can pass it a function for custom validation.

Reply:
I was trying to write the tests for `referee` not using `referee` itself.

Reply:
I'm talking about the node assert (see link).

Reply:
Oh! I WILL check that out @khamiltonuk

Reply:
I have added a commit that uses `assert.throws`. That looks much better!